api/envoy/config/cluster/v3/outlier_detection.proto

```diff
@@ -173,4 +173,8 @@ message OutlierDetection {
   // Set of host's passive monitors.
   // [#not-implemented-hide:]
   repeated core.v3.TypedExtensionConfig monitors = 24;
+
+  // If enabled, :ref:`base_ejection_time<envoy_v3_api_field_config.cluster.v3.OutlierDetection.max_ejection_percent>` is not considered for first ejection.
```

Comment:
I think that this comment is a bit confusing. The first time I read it, I thought that "first ejection" means the first time outlier detector is run. Maybe it should be changed to something:

`If enabled, at least one host is ejected, regardless of the value of :ref:`base_ejection_time<envoy_v3_api_field_config.cluster.v3.OutlierDetection.max_ejection_percent>`.`